src/Target.cpp

```diff
@@ -422,7 +422,7 @@ Target get_jit_target_from_environment() {
     host.set_feature(Target::TSAN);
 #endif
 #if __has_feature(coverage_sanitizer)
-    host.set_feature(Target::SANCOV);
+    host.set_feature(Target::SANITIZERCOVERAGE);
```

Comment:
This looks like it won't compile, did you mean Target::SanitizerCoverage

Reply:
I did verify that this builds for me, starting with clean build dir.